en/user_guide/node.rst

```diff
@@ -0,0 +1,86 @@
+Node
+====
+
+Languages
+---------
+
+A node has several languages defined during the configuration of Back Office.
+The node language being edited can be changed by clicking on the language tab. If this node has no data for this language, it will be created.
+
+.. image:: ../../images/languages.png
+
+Workflow
+--------
+
+Nodes have a status and can have a validation workflow, which is the series of steps to achieve for publication of a node before you can display it on Front Office.
+Status of a node is linked to language of this one.
+Node have an initial status for this creation and to be able to display on Front Office the node has to have a final status.
+
+.. image:: ../../images/workflow.png
+
+To know more about status see also `workflow`_.
+
+Versionning
+-----------
+
+A node have several versions.
+The node version is linked to language of this one.
+To create a new version of a node click on the button "new version".
+
+.. image:: ../../images/new_version.png
+
+The latest version of the node is duplicated and the node is set to in the initial status.
+
+Then find all the version of a node in the drop-down list.
+
+.. image:: ../../images/version_list.png
+
+The latest version of the node in the final status will be the one displayed in Front Office.
+
+Blocks
+------
+
+Open Orchestra allows to add blocks in nodes.
+The visible list of blocks during the edition of a node is defined during the site configuration.
+
+See also `node configuration`_ and `site configuration`_.
+
+There are two types of blocks :
+* Transverse block.
+* Not transverse block.
+
+See also `node and block transverse`_.
+
+The list of blocks is in the blocks tab, which is visible only if the node is alterable (not in the final status).
+
+.. image:: ../../images/right_panel.png
+
+See also `block list`_.
+
+Drag&Drop
+~~~~~~~~~
+
+Open Orchestra allows to add or to move blocks in a node using drag and drop.
+At the top of the list of block there are the new blocks and below transverse blocks. Modify transverse blocks in the transverse node.
```

Comment:
"Modify transverse blocks in the transverse node." => I'm not sure that this point should be mentionned in the drag'n'drop section.